packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -169,46 +171,236 @@ export const ShopperBasketsMutations = {
     AddTaxesForBasket: 'addTaxesForBasket'
 } as const
 
-type ShopperBasketMutationType = typeof ShopperBasketsMutations[keyof typeof ShopperBasketsMutations]
+export type ShopperBasketMutationType = typeof ShopperBasketsMutations[keyof typeof ShopperBasketsMutations]
+
+export const getQueryKeysMatrix = (customerId: string | null) => {
+    const updateBasketQuery = (basketId?: string) => {
+        // TODO: we're missing headers, rawResponse -> not only {basketId}
+        const arg = {basketId}
+        return basketId
+            ? {
+                  update: [['/baskets', basketId, arg]]
```

Comment:
If I write an inaccurate query key like this: `update: [['/baskets', basketId, 'FOO', arg]]`
some tests would fail as expected → this is possible because we're testing the mutation hooks against the query hooks, as I mentioned in my previous comment.

If I do the same thing to a query key in the other file for ShopperCustomers, all the tests there would still pass.